src/js/media-container.js

```diff
@@ -47,37 +63,51 @@ template.innerHTML = `
       background: none;
     }
 
-    ${/*
-     * when in audio mode, hide the gesture-layer which causes media-controller to be taller than the control bar
-     */''}
-    :host([audio]) [part~=layer][part~=gesture-layer] {
+    ${
+      /*
+       * when in audio mode, hide the gesture-layer which causes media-controller to be taller than the control bar
+       */ ''
+    }
+    :host([${Attributes.AUDIO}]) [part~=layer][part~=gesture-layer] {
       height: 0;
       display: block;
     }
 
-    ${/*
-     * if gestures are disabled, don't accept pointer-events
-     */''}
-    :host(:not([audio])[gestures-disabled]) ::slotted([slot=gestures-chrome]),
-    :host(:not([audio])[gestures-disabled]) media-gesture-receiver[slot=gestures-chrome] {
+    ${
+      /*
+       * if gestures are disabled, don't accept pointer-events
+       */ ''
+    }
+    :host(:not([${Attributes.AUDIO}])[${
```

Comment:
oof formatter really did a number on this one. Will unmangle.

Reply:
yessir!

Reply:
which formatter did this? `yarn format`?

Reply:
I think it was some default prettier line length creeping in? Haven't investigated.